float64 x
float64 y
string map_name
---
bool success
string message
---

